### DISCUSSION TITLE
Next js 13 - Hydration failed because the initial UI does not match... Problem

### DISCUSSION THREAD

**Discussion by Roma79794** [vercel/next.js#45274] [Help]
I get this error when I focus on the input element as soon as the page opens. I don't get an error when I wait a few seconds and focus on the input element. I can't solve this error. When I remove the Link components the error is resolved.

Just to be sure, I installed the LATEST DEFAULT NEXT JS 13.1.5 project and added the input element directly, the same problem continues. [Error image](https://i.ibb.co/48WcFPq/2023-01-25-22-08-16.png)

[Error detailed image](https://i.stack.imgur.com/vVVqt.png)

`

        {/* When i deleted this div section problem is fixed */}
        <div className="flex gap-x-11">
          <Link className="font-medium" href="/asdasd">
            Bütün məhsullar (133)
          </Link>
          <Link className="text-black/50" href="/asdasd">
            Populyar
          </Link>
          <Link className="text-black/50" href="/asdasd">
            Yeni
          </Link>
          <Link className="text-black/50" href="/asdasd">
            Endirimli
          </Link>
        </div>
        {/* When i deleted this div section problem is fixed */}

        <input type="text" placeholder="text" />
`

**Comment by icyJoseph** [comment]
Ah I recall something similar from the early days of Next 13. The problem then was that when you click on the page immediately, React drops the hydration it's currently doing and focuses on hydrating the part where the user clicked. For some reason that causes the hydration error message to bubble up.

**Reply by Roma79794** [reply]
Yes exactly. I tried and the problem is not the input focus. I get this error when I click somewhere on the page while the page is loading. =(  Is there a solution? Thanks.
